packages/tlschema/src/records/TLUser.ts

```diff
@@ -52,12 +52,12 @@ export const TLUser = createRecordType<TLUser>('user', {
 	let lang
 	if (typeof window !== 'undefined' && window.navigator) {
 		const availLocales = LANGUAGES.map(({ locale }) => locale) as string[]
-		lang = window.navigator.languages.find((lang) => {
-			return availLocales.indexOf(lang) > -1
-		})
+		lang = window.navigator.languages.find(
+			(lang) => availLocales.indexOf(lang.split(/[-_]/)[0]) > -1
```

Comment:
here lang would be `'en-US'` but the "availLocales" would be `[`en`, `fr`]` etc.

Reply:
note that we WANT to return `'en-US'`, not just `'en'`.

Reply:
I don't think this is what we want either. That only works sometimes, what about `zh-cn` and `zh-tw`

```js
	{ locale: 'zh-cn', label: 'Chinese - Simplified' },
	{ locale: 'zh-tw', label: '繁體中文 (台灣)' },
```

I think what you want is expand `packages/tlschema/src/languages.ts` to be a single data-type of proper ico-code with country. Or have a fallback list

Reply:
Damn, true. I don't know what the complete answer is here. Since we're doing a lookup, we could try the full locale first and then just language?

Reply:
Yeah that's what I meant by _fallback_ above. Otherwise just load the complete list of iso codes https://github.com/TiagoDanin/Locale-Codes/tree/master probably a pretty small list. I'd go with option 1 though

Reply:
I'd probably store 2 lists of languages, the `packages/tlschema/src/languages.ts` should probably be the full code as that's what it is, I don't know what spelling `en` would be referring to otherwise, pinging @TodePond for clarification.  

Then we can have a fallback list, so `en` -> `en-US` for example, and do the `.split('-')[0]` if the above doesn't match.

Reply:
I've seen the latest commit @steveruizok. For clarification shouldn't `en` be `en-GB` in `packages/tlschema/src/languages.ts` because I don't know what `en` refers to, is is `color` or `colour`. I think we should be specifying our locale fully, maybe?? I guess Lu will have context on this from the translations work.

This was what I meant by a fallback list, essentially

```js
const fallback = {'en': 'en-US'}
```

I assume @TodePond will have the context here.